### PR TITLE
framework/ble_manager: add ble mac filter param

### PR DESCRIPTION
**Pull request by an4967**
- add ble mac filter param

### CODE REVIEW

File: framework/include/ble_manager/ble_client.h [hunk 1]
```diff
@@ -83,6 +83,7 @@ typedef struct {
 typedef struct {
 	uint8_t raw_data[BLE_ADV_RAW_DATA_MAX_LEN];
 	uint8_t raw_data_length;
+	uint8_t mac[BLE_BD_ADDR_MAX_LEN];
```
Comment (jeongchanKim) [inline framework/include/ble_manager/ble_client.h:86]: When will this be used?

Comment (an4967) [reply framework/include/ble_manager/ble_client.h:86]: This is a format for implementing BLE driver.
After 'BLE drirver' works done, I will make a new example.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by pillip8282 — APPROVED** [review]
lgtm

**Pull request merged**